src/index.ts

```diff
@@ -2,6 +2,11 @@ import { parseOptions } from './Options';
 import { runWithServer } from './Server';
 import { runSuite } from './Suite';
 
+export type AxeParams = {
+  skip?: boolean;
+  disabledRules?: string[];
+  timeout?: number;
```

Comment:
Would it make sense to add jsdoc comments to these parameters, to provide some more info to people seeing these in an app?

For example, I could see it being useful to:
- Point out that `disabledRules` should the name of an axe-core rule from https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md.
- Point out that `timeout` should be in milliseconds
- etc